common/components/WalletDecrypt/WalletDecrypt.tsx

```diff
@@ -213,7 +214,8 @@ const WalletDecrypt = withRouter<Props>(
     public state: State = {
       selectedWalletKey: null,
       isInsecureOverridden: false,
-      value: null
+      value: null,
+      walletTimeoutID: undefined
```

Comment:
@Sonicrida 
The component doesn't utilize walletTimeoutId as part of it's visual output, so it can be an instance variable, `this.walletTimeoutID`. Also, default it to `null`; `undefined` is semantically better for values that you haven't defined -- defining an `undefined`.